packages/account/src/Components/poi/poi-form-on-signup/idv-doc-submit-on-signup/idv-doc-submit-on-signup.tsx

```diff
@@ -29,13 +29,15 @@ type TIdvDocSubmitOnSignup = {
     has_idv_error?: boolean;
     account_settings: GetSettings;
     getChangeableFields: () => string[];
+    residence_list: any;
```

Comment:
lets use TResidenseList type insted of any

Reply:
oh, thanks for noticing it @amina-deriv,
I think we could you better ResidenceList coming from @deriv/api-types

Reply:
Let’s get red of custom types and use those from api